src/Umbraco.Web.BackOffice/Controllers/MemberController.cs

```diff
@@ -55,6 +56,7 @@ public class MemberController : ContentControllerBase
     private readonly ITwoFactorLoginService _twoFactorLoginService;
     private readonly IShortStringHelper _shortStringHelper;
     private readonly IUmbracoMapper _umbracoMapper;
+    private readonly SecuritySettings? _securitySettings;
```

Comment:
I don't see this field being set anywhere, so it ends up always being `null` and not applying the correct setting: have you (re-)tested creating a member using a duplicate e-mail address?

I'd also recommend using `IOptionsMonitor<SecuritySettings>`, so any updates to the configuration is always used (since controllers are registered as singletons). Also keep the constructor updates/obsoletion recommendations in mind when fixing this!